travis/setup_conda.sh

```diff
@@ -47,7 +47,7 @@ if [[ ! -z $(echo ${COMMIT_MESSAGE} | grep -E "${TR_SKIP}") ]]; then
     echo "Travis was requested to be skipped by the commit message, exiting."
     travis_terminate 0
 elif [[ ! -z $(echo ${COMMIT_MESSAGE} | grep -E "${DOCS_ONLY}") ]]; then
-    if ! [[ $SETUP_CMD =~ build_docs|build_sphinx|pycodestyle|flake8|pep8 ]] && [[ MAIN_CMD =~ pycodestyle|flake8|pep8 ]]; then
+    if [[ ! $SETUP_CMD =~ build_docs|build_sphinx|pycodestyle|flake8|pep8 ]] && [[ ! $MAIN_CMD =~ pycodestyle|flake8|pep8 ]]; then
```

Comment:
Is it better like this?

Reply:
I prefer the other way as it's clearer what to negate, but I'm ok with this way, too if it works. (e.g. I'm not sure about the precedent order for `!` vs `=~`).

Reply:
I think the CI for this PR is skipping the right stuff so far? Though I can't tell whether the doc build log means it is skipped or just it found nothing to build. The message difference is very subtle to me.

Reply:
I'll have a look. Basically for the skipped jobs there shouldn't be any dependencies installed, the skip should happen before the script goes into parsing and installing anything.

Reply:
It indeed seems to work, but could you also add `pylint` to this list?

Reply:
Added. Also skipped Appveyor this time since this PR does not touch Appveyor.

Reply:
Ops... forgot the `[docs only]` in second commit. Do you want me to add it and force push?

Reply:
yeah, please let's do that otherwise we get to our old age when it finishes (actually could also do a [skip ci] as I'm prepared to live dangerously and override the branch protection, as it already passed before).

Reply:
Re-pushed!